core/ic/src/tera/src/api/messages.rs

```diff
@@ -7,9 +7,19 @@ use crate::{
     tera::STATE,
 };
 
+const MAX_REMOVE_MESSAGES: usize = 1000;
```

Comment:
we can prob increase this to way more than 1k. As long as it doesn't exceed the 2MB limit.

Reply:
Done 👍